creation/host-devices.md

```diff
@@ -0,0 +1,59 @@
+Host Devices Assignment
+=======================
+
+KubeVirt provides a mechanism for assigning host devices to a virtual machine.
+This mechanism is generic and allows various types of PCI devices, such as GPU
+or any other devices attached to a PCI bus,  to be assigned. It also allows
+Mediated devices, such as pre-configured virtual GPUs to be assigned using the
+same mechanism.
+  
+
+Listing Permitted Devices
+-------------------------
+
+Administrators can control which host devices are permitted to be used in the
+cluster.
+Permitted host devices in the cluster will need to be whitelisted in KubeVirt
+CR by its vendor:product selector for PCI devices or mediated device names 
+
+```
+configuration:
+  permittedHostDevices:
+    pciHostDevices:
+    - pciVendorSelector: "10DE:1EB8"
+      resourceName: "nvidia.com/TU104GL_Tesla_T4"
+      externalResourceProvider: true
+    - pciVendorSelector: "8086:6F54"
+      resourceName: "intel.com/QAT"
+    mediatedDevices:
+    - mdevNameSelector: "GRID T4-1Q"
+      resourceName: "nvidia.com/GRID_T4-1Q"
+```
+
```

Comment:
Please add small sections to explain how pciVendorSelector, mdevNameSelector works.

Reply:
Done